Project/core/src/main/java/Controller/GameController.java

```diff
@@ -163,7 +163,39 @@ public void newTurn() {
         if (checkAction.isMaskEmpty(gameState.getValidityMask())) {
             gameState.setGameOver();
         }
+        if (gameState.getCurrentPlayer() instanceof Recruiter) {
+            setVisibility(true);
+        } else {
+            setVisibility(false);
+        }
+    }
 
+    /**
+     * Sets the visibility of all steps for the recruiter
+     * 
+     * @param visibility what to set visibility to
+     */
+    private void setVisibility(boolean visibility) {
```

Comment:
Maybe this method could be called `setRecruiterVisibility` to emphisize that it is related to the recruiters turn